Duplicati/UnitTest/SymLinkTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Duplicati.Library.Interface;
+using Duplicati.Library.Main;
+using Duplicati.Library.Snapshots;
+using NUnit.Framework;
+
+namespace Duplicati.UnitTest
+{
+    [TestFixture]
+    public class SymLinkTests : BasicSetupHelper
+    {
+        [Test]
+        [Category("SymLink")]
+        public void SymLinkPolicy()

```

Comment:
What are your thoughts about simplifying this by parameterizing the test with
```
[Test]
[Category("SymLink")]
[TestCase(Options.SymlinkStrategy.Store)]
[TestCase(Options.SymlinkStrategy.Follow)]
[TestCase(Options.SymlinkStrategy.Ignore)]
public void SymLinkPolicy(Options.SymlinkStrategy symlinkPolicy)
```
We can then remove the 
```
foreach (var symlinkPolicy in symlinkPolicies)
```
and remove all the file and directory deletion statements as well.

This also improves the test reporting by treating each parameter as a separate test.

Reply:
That's very nice!  I've pushed a commit with this change.